tests/test_path.py

```diff
@@ -8,18 +8,19 @@
 from scantree.test_utils import assert_dir_entry_equal
 
 
-def create_basic_entries(local_path):
+def create_basic_entries(local_path: str):
```

Comment:
Not a `str` see previous comment. (Would be great with type annotations, but in a separate PR :))

Reply:
reverted